admin/admin-service/src/test/java/io/mosip/admin/bulkdataupload/batch/CustomExcelRowMapperTest.java

```diff
@@ -0,0 +1,184 @@
+                .createBinder("Target");
+        assertFalse(actualCreateBinderResult.isIgnoreUnknownFields());
+        assertFalse(actualCreateBinderResult.isIgnoreInvalidFields());
+        assertTrue(actualCreateBinderResult.isAutoGrowNestedPaths());
+        assertNull(actualCreateBinderResult.getValidator());
+        assertEquals("Target", actualCreateBinderResult.getTarget());
+        assertEquals("target", actualCreateBinderResult.getObjectName());
+        assertTrue(actualCreateBinderResult.getBindingErrorProcessor() instanceof DefaultBindingErrorProcessor);
+        assertEquals(256, actualCreateBinderResult.getAutoGrowCollectionLimit());
+    }
+
+    @Test
+    public void createBinder_withDefaultFormattingConversionService_returnSuccessResponse() {
+        DefaultFormattingConversionService conversionService = new DefaultFormattingConversionService(
+                mock(StringValueResolver.class), true);
+
+        DataBinder actualCreateBinderResult = (new CustomExcelRowMapper<>(conversionService, new CustomValidatorBean()))
+                .createBinder("Target");
+        assertFalse(actualCreateBinderResult.isIgnoreUnknownFields());
+        assertFalse(actualCreateBinderResult.isIgnoreInvalidFields());
+        assertTrue(actualCreateBinderResult.isAutoGrowNestedPaths());
+        assertNull(actualCreateBinderResult.getValidator());
+        assertEquals("Target", actualCreateBinderResult.getTarget());
+        assertEquals("target", actualCreateBinderResult.getObjectName());
+        assertTrue(actualCreateBinderResult.getBindingErrorProcessor() instanceof DefaultBindingErrorProcessor);
+        assertEquals(256, actualCreateBinderResult.getAutoGrowCollectionLimit());
+    }
+
+    @Test
+    public void findPropertyName_withValidRequest_thenPass() {
+        ApplicationConversionService conversionService = new ApplicationConversionService();
+
+        Object bean = "Bean";
+        String key = "Key";
+
+        ReflectionTestUtils.invokeMethod(new CustomExcelRowMapper<>(conversionService, new CustomValidatorBean()),
+                "findPropertyName", bean, key);
+
+    }
+
+    @Test
+    public void findPropertyName_withNullRequest_returnNull() {
+        ApplicationConversionService conversionService = new ApplicationConversionService();
+
+        ReflectionTestUtils.invokeMethod(new CustomExcelRowMapper<>(conversionService, new CustomValidatorBean()),
+                "findPropertyName", null, null);
+
+    }
+
+    @Test
+    public void findPropertyName_withNullBean_returnNull() {
+        ApplicationConversionService conversionService = new ApplicationConversionService();
+
+        Object bean = null;
+        String key = "Key";
+
+        ReflectionTestUtils.invokeMethod(new CustomExcelRowMapper<>(conversionService, new CustomValidatorBean()),
+                "findPropertyName", bean, key);
+    }
+
+    @Test
+    public void CustomExcelRowMapper_withSetValues_returnSuccessResponse() {
```

Comment:
Method name donot start with capital letter, kindly adhere to method naming convention.

Reply:
Changed the method name by following the naming convention